src/stories/Library/opening-hours/opening-hours.scss

```
@@ -0,0 +1,89 @@
+$_nav-control-height: 40px;
+$_nav-control-border-width: 2px;
+$_opening-hours-max-width: 600px;
+
+.opening-hours {
+  @include layout-container;
+}
+.opening-hours__wrapper {
+  max-width: $_opening-hours-max-width;
+  margin-right: auto;
+}
+
+.opening-hours__header {
+  display: flex;
+  justify-content: space-between;
+  flex-direction: column;
+  gap: $s-lg;
+  margin-bottom: $s-lg;
+  align-items: center;
+
+  @include media-query__small {
+    flex-direction: row;
+  }
+}
+
+.opening-hours__heading {
+  @include typography($typo__h3);
+  align-self: flex-start;
+}
+
+.opening-hours__row {
+  padding: $s-md 0 $s-lg 0;
+  border-bottom: 1px solid $color__global-tertiary-1;
+  @include typography($typo__body-placeholder);
+}
+
+.opening-hours__individual-day {
+  @include typography($typo__body-medium);
+  font-weight: $font__weight--bold;
+  margin-bottom: $s-md;
+  @include media-query__small {
+    @include typography($typo__body-large);
+    font-weight: $font__weight--bold;
+  }
+}
+
+.opening-hours__individual-opening {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 $s-xs;
+  @include media-query__small {
+    line-height: 32px;
+  }
+}
+.opening-hours__individual-opening--odd {
```

```suggestion
.opening-hours__individual-opening:nth-child(odd)  {
```

I think this can be simplified a bit using [`:nth-child()`](https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child#trnth-childodd) like we use `:first-child` below.

This means we have remove the logic for applying this class also.

---

Yeah this is better 👍